src/Process/Runner/InitialTestsRunner.php

```diff
@@ -66,16 +71,35 @@ public function run(
         array $phpExtraOptions,
         bool $skipCoverage
     ): Process {
+        /** @var Process $process */
         $process = $this->processBuilder->createProcess(
             $testFrameworkExtraOptions,
             $phpExtraOptions,
             $skipCoverage
         );
 
+        //Tracking Process Error Exit
```

Comment:
CS nitpick, we used to leave a space between the `//` and the text, i.e. `// Tracking`

Reply:
Updated